packages/ragbits-core/src/ragbits/core/llms/litellm.py

```diff
@@ -83,3 +85,24 @@ def count_tokens(self, prompt: BasePrompt) -> int:
             Number of tokens in the prompt.
         """
         return sum(litellm.token_counter(model=self.model_name, text=message["content"]) for message in prompt.chat)
+
+    def _format_chat_for_llm(self, prompt: BasePrompt) -> ChatFormat:
```

Comment:
`ChatFormat` defined as `list[dict[str, str]]` is not the correct type here - this causes you to have to sprinkle `# type: ignore` throughout the documentation, but more importantly is not a true representation of what this method returns. Either use a separate type defined as `list[dict[str, Any]]` here, or change the `ChatFormat` definition.

Reply:
changed